Project-Descriptions-and-Plans/CVA6/CVA6-main-plan-approved-2024+.md

```diff
@@ -0,0 +1,524 @@
+
+| Related TG      | Milestone                                                                                                                      | Target        | Contributor                                   |
+| --------------- | ------------------------------------------------------------------------------------------------------------------------------ | ------------- | ------------------------------------------    |
+| Project-wide    | Customize the release process for CVA6                                                                                         | 2024-Q1       | Project and sub-project leaders, OpenHW staff |
+| Project-wide    | Review proposed CVA6 evolutions                                                                                                | Continuous    | All, coordinated by Jérôme                    |
+| Project-wide    | Maintain and upgrade User Manual                                                                                               | Continuous    | All, coordinated by Jérôme                    |
+| Project-wide    | Maintain and upgrade Design Document                                                                                           | Continuous    | All, coordinated by Jean-Roch                 |
+| Project-wide    | Triage of PR (including 2<sup>nd</sup> level reviews from sub-projects) and issues                                             | Continuous    | Jean-Roch, with Mike's support                |
+| Project-wide    | Reviews of PR (including 2<sup>nd</sup> level reviews from sub-projects), handling of issues                                   | Continuous    | As per triage                                 |
+| Cores           | Unify three MMU (Sv32, Sv39, Sv39x4) in a single source code, as a prerequisite to the hypervisor mode integration             | 2024-Q1       | PlanV                                         |
+| Cores           | Integrate the hypervisor mode to the master branch                                                                             | 2024-Q2       | Zero-Day Labs, with TSS support               |
+| Cores           | Add support of Microchip PolarFire FPGA (port FPGA optimizations)                                                              | 2024          | TRT                                           |
+| Cores           | Add support of Microchip PolarFire FPGA (APU adaptation)                                                                       | 2024          | Undisclosed new member                        |
+| Cores           | Add support of Intel FPGA (port FPGA optimizations, APU adaptation)                                                            | 2024          | TRT                                           |
+| Cores           | Define and add memory protection for harsh environments (error detection and correction)                                       | 2024          | Interested stakeholders                       |
```

Comment:
ditto comment above - sounds interesting - where is it described?

Reply:
https://drive.google.com/drive/folders/10tafeJ2m0nBxH8FkINz2FdK2aRe3GSJP
At the CVA6 workshop in Marseille, several stakeholders realized they have the same need. Better team up.